tests/notification/test_sites.py

```diff
@@ -49,3 +56,44 @@ def test_multiple_added_sites_are_reflected_by_on_site(self):
         self.assertIn(notification, EmailNotification.objects.on_site())
         with TempSiteID(site_2.id):
             self.assertIn(notification, EmailNotification.objects.on_site())
+
+
+class SiteRequestTest(TestCase):
+
+    def setUp(self):
+        super(SiteRequestTest, self).setUp()
+        User.objects.create_user(username='dummy', password='dummy')
+        self.client.login(username='dummy', password='dummy')
+        user = User.objects.get(username='dummy')
+        self.report = Report(owner=user)
+        self.report_key = 'bananabread! is not my key'
+        self.report.encrypt_report('{}', self.report_key)
+        self.report.save()
+        self.submit_url = reverse('test_submit_report', args=[self.report.pk])
+
+    @override_settings()
+    @patch('django.http.request.HttpRequest.get_host')
+    def test_can_request_pages_without_site_id_set(self, mock_get_host):
```

Comment:
I wrote this test to catch https://github.com/SexualHealthInnovations/callisto-core/pull/151/commits/1b5c60969b3d021994526d18b3dea613b00ccf67

then decided it wouldn't hurt to leave it in